scalableinternetservices/team-temp
# Add functionalities to webpage

Including:
- hook 'add question' button to actual url
- create a `session` data model for user login
- add a 'login' button in the home page
- add a 'sign up' html page and leave the link under login page
- fix end line format in Langhao's PR


What I did to test the functionalities:
- run `rails server` should launch the home webpage
- it has two main buttons: login and add question
- click on login will redirect to /sessions/new
- in the login page, click on sign up will redirect to /user/new

[tested]
